server/src/main/scala/wop/server/actors/GameActor.scala

```diff
@@ -24,7 +27,24 @@ class GameActor(playerA: ActorRef, playerB: ActorRef, aName: String, bName: Stri
   playerA ! PlayerActor.EnterGame(self, state, playersMap(playerA), bName)
   playerB ! PlayerActor.EnterGame(self, state, playersMap(playerB), aName)
 
+  case object Tick
+
+  context.system.scheduler.schedule(1 second, 1 second) {
```

Comment:
Sometime we can see laggy timer cause latency. May be we should tick every 0.5 seconds.